models/article-model.js

```diff
@@ -36,4 +36,19 @@ function responseComments(article_id) {
     })
 }
 
-module.exports = { responseAllArticles, responseArticle, responseComments }
\ No newline at end of file
+function insertComment({ user_name, body }, article_id) {
+    if (!user_name) {
+        return Promise.reject({
+            status: 400, msg: 'Bad request'
+        })
+    }
+    return db.query(`ALTER TABLE comments DROP CONSTRAINT comments_author_fkey`)

```

Comment:
We shouldn't need to do this, the INSERT SQL query below should be all we need 👍